src/middlewared/middlewared/plugins/service_monitor.py

```diff
@@ -125,16 +133,29 @@ def run(self):
         ntries = 0
         service = self.name
 
+        if service == 'activedirectory':
+            fnldap = FreeNAS_ActiveDirectory(flags=FLAGS_DBINIT)
+        elif service == 'ldap':
+            fnldap = FreeNAS_LDAP(flags=FLAGS_DBINIT)
+        else:
+            fnldap = None
```

Comment:
Minor thing, but you can just declare above `fnldap = None` and avoid having `else:` clause.